frontend/src/components/Sidebar.tsx

```diff
@@ -336,9 +338,15 @@ function VersionButton() {
       <Button
         onClick={() => setOpen(true) }
         style={{textTransform: 'none', }}
-        startIcon={<Icon color="#adadad" icon={kubernetesIcon} className={classes.versionIcon} />}
       >
-        {clusterVersion.gitVersion}
+        <Box display={sidebar.isSidebarOpen ? 'flex' : 'block'} alignItems="center">
+          <Box>
+            <Icon color="#adadad" icon={kubernetesIcon} className={classes.versionIcon} />
+          </Box>
+          <Box>
+            {clusterVersion.gitVersion}
+          </Box>
+        </Box>
```

Comment:
It would be great if we could just rely on CSS instead of having to use the sidebar-open logic, etc.
Maybe using a Grid?

Reply:
Let's keep this for now then.